Game/Src/Characters/StatsGen.gd

```diff
@@ -1,16 +1,16 @@
 extends Node
 
 class_name StatsGen
-
-export var max_health = 0.2
-export var strength = 10
-export var defense = 5
-
-onready var health = max_health setget set_health
-
-signal no_health
-
-func set_health(value):
-	health = value
-	if health <= 0:
-		emit_signal("no_health")
+#
```

Comment:
Why put everything in comments? Can we delete it instead?

Reply:
I just commented a ton for legacy reasons while testing new implementations. It was easier to go back if any trouble was met. Maybe some cleanup would be welcome :)

Reply:
Actually, did not pay attention, no clue what this script is